grails-app/migrations/0.9.x/changelog-2024-07-18-1200-alter-table-location-type-make-location-type-code-non-null.groovy

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog = {
+
+    changeSet(author: "ewaterman", id: "1721321391197-94", objectQuotingStrategy: "LEGACY") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            and {
+                columnExists(columnName: "location_type_code", tableName: "location_type")
+
+                sqlCheck("""SELECT COUNT(*) FROM location_type WHERE location_type_code IS NULL""", expectedResult: "0")
```

Comment:
I think there should be just one quote instead of three.

Reply:
agreed, I originally had this on multiple lines (hence the triple quotes) but then condensed it to one so the triple quote isn't needed anymore